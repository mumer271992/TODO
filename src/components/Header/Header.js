import React from 'react';
import { NavLink } from 'react-router-dom';

import './Header.scss';

const Header = () => {
  return (
    <nav className="nav nav-pills nav-fill">
      <NavLink data-test="link-all" activeClassName="active" className="nav-item nav-link" to="/home">All</NavLink>
      <NavLink data-test="link-completed" activeClassName="active" className="nav-item nav-link" to="/completed">Completed</NavLink>
      <NavLink data-test="link-trash" activeClassName="active" className="nav-item nav-link" to="/trash">Trash</NavLink>
      <NavLink data-test="link-create" activeClassName="active" className="nav-item nav-link" to="/todo">Create</NavLink>
    </nav>
  );
};

export default Header;
