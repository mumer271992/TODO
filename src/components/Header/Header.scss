.nav{
  border: 1px solid rgba(0,0,0,.125);
  margin-bottom: 2rem;
  background-color: #ffffff;
  .nav-link{
    padding: 1.2rem 1rem;
    border-radius: 0px;
  }
}