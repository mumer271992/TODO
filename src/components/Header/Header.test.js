import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('Header component unit tests.', () => {
  it('Component should render successfully', () => {
    const shallowWrapper = shallow(<Header />);
    expect(shallowWrapper.length).toEqual(1);
    expect(shallowWrapper.wrapper).toMatchSnapshot();
  });
  // TODO: write unit tests for navigation to all pages through header tabs
});
