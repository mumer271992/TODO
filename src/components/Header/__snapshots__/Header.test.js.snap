// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component unit tests. Component should render successfully 1`] = `undefined`;
